app/org/sagebionetworks/bridge/models/SynapseAccount.java

```diff
@@ -0,0 +1,49 @@
+package org.sagebionetworks.bridge.models;
+
+/**
+ * Represent a Synapse Account
+ */
+public class SynapseAccount {
+    private String email;
+    private String username;
+    private String firstName;
+    private String lastName;
+
+    public boolean isValid() {
+        return email != null && username != null && firstName != null && lastName != null;
+    }
```

Comment:
As far as I can tell, Synapse users are not required to have first name or last name. Do we need to validate that they exist?

Also, for validation, you should use the pattern set in the validators package in BridgePF.

Third, what about empty strings or blank strings?

Lastly, what unit tests do we have on validation?